src/HDInsight/HDInsight.Test/UnitTests/HttpTests.cs

```diff
@@ -43,15 +47,26 @@ public HttpTests(Xunit.Abstractions.ITestOutputHelper output)
                 ResourceGroupName = ResourceGroupName,
                 HttpCredential = _httpCred
             };
+
             revokecmdlet = new RevokeAzureHDInsightHttpServicesAccessCommand
             {
                 CommandRuntime = commandRuntimeMock.Object,
                 HDInsightManagementClient = hdinsightManagementMock.Object,
                 ClusterName = ClusterName,
                 ResourceGroupName = ResourceGroupName
             };
+
+            updatecmdlet = new SetAzureHDInsightGatewayCredentialCommand
+            {
+                CommandRuntime = commandRuntimeMock.Object,
+                HDInsightManagementClient = hdinsightManagementMock.Object,
+                ClusterName = ClusterName,
+                ResourceGroupName = ResourceGroupName,
+                HttpCredential = _httpCred
+            };
         }
 
+        [Obsolete]
```

Comment:
Please do not mark tests as obsolete.  If these tests are actually not useful, please remove them.

Reply:
> Please do not mark tests as obsolete. If these tests are actually not useful, please remove them.

fixed

Reply:
fixed

---
In reply to: [278674096](https://github.com/Azure/azure-powershell/pull/9069#discussion_r278674096) [](ancestors = 278674096)

Reply:
fixed

---
In reply to: [279183702](https://github.com/Azure/azure-powershell/pull/9069#discussion_r279183702) [](ancestors = 279183702)